uploader/models.py

```diff
@@ -1,3 +1,15 @@
 from django.db import models
 
-# Create your models here.
+from django.utils import timezone
+
+
+class FileUpload(models.Model):
+
+    number_of_files = models.PositiveIntegerField("Number of Files Uploaded", default=0)
+    username = models.CharField("Username", max_length=80)
+    uploaded_at = models.DateTimeField("Uploaded at", auto_now=True, editable=False)
+
+    def __str__(self):
+        return "{}  :  {} : {})".format(
```

Comment:
aha, I see we are on python 3.5 and dont have access to f strings. Well, in that case we might want to use numbered arguments, `"{0}: {1}: {2}".format()`

Reply:
Sure!